staging/src/k8s.io/client-go/transport/cache.go

```diff
@@ -44,7 +43,7 @@ var DialerStopCh = wait.NeverStop
 
 const idleConnsPerHost = 25
 
-var tlsCache = &tlsTransportCache{transports: make(map[tlsCacheKey]*http.Transport)}
+var tlsCache = &tlsTransportCache{transports: cache.NewLRUExpireCache(256)}
```

Comment:
can we define them as constants and set some large numbers like 8096 to be on the safe side?

https://github.com/kubernetes/kubernetes/pull/85957